homeassistant/components/nextcloud/config_flow.py

```diff
@@ -29,6 +43,12 @@ class NextcloudConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    def __init__(self) -> None:
+        """Initialize flow."""
+        self._entry: ConfigEntry | None = None
+        self._url: str | None = None
+        self._username: str | None = None
+
```

Comment:
I think only the config entry is needed really, and it could be defined as a class attributes
```suggestion
        _entry: ConfigEntry | None = None

```